### ISSUE TITLE
What's the meaning of tw and th in your loss function?

### ISSUE THREAD

**Issue by XiaXuehai** [eriklindernoren/PyTorch-YOLOv3#49]
When you calculate loss, the code about  `tw` and `th` like this:
```
# Width and height
tw[b, best_n, gj, gi] = math.log(gw/anchors[best_n][0] + 1e-16)
th[b, best_n, gj, gi] = math.log(gh/anchors[best_n][1] + 1e-16)
```
I don't understand why you use ` log`,and in the paper, the loss function about w and h is 
![image](https://user-images.githubusercontent.com/11585321/42499373-2e82e3dc-8461-11e8-9a1a-3f190830b531.png)
And why loss_x and loss_y use BCELoss? It's a regression problem,I think MSELoss is applicable.
Can you explain it ? THX.

**Comment by XiaXuehai** [comment]
I have known it.

**Comment by debiff** [comment]
@XiaXuehai Please, can you explain it?
